util/network-devp2p/src/discovery.rs

```diff
@@ -1096,14 +1202,6 @@ mod tests {
 			panic!("Expected discovery1 to be added to discovery1's table");
 		}
 
-		let ping_back = discovery2.dequeue_send().unwrap();
-		assert!(!discovery2.any_sends_queued());
-		let data = &ping_back.payload[(32 + 65)..];
-		assert_eq!(data[0], PACKET_PING);
-		let rlp = Rlp::new(&data[1..]);
-		assert_eq!(ep2, NodeEndpoint::from_rlp(&rlp.at(1).unwrap()).unwrap());
-		assert_eq!(ep1, NodeEndpoint::from_rlp(&rlp.at(2).unwrap()).unwrap());
```

Comment:
On `Ping` packet received, the node that sent the request is now added to the NodeTable without going through another `Ping/Pong` round.